### PR TITLE
DTFS2-4727 - only call estore if BSS

### PR DESCRIPTION
**Pull request by ttbarnes**
- remove incomplete exporters from GEF data. Can cause issues with estore
- in GEF submit-to-ukef e2e test, use a deal with only 1 facility
- disable estore API call until it doesn't error (pending Mulesoft IT team)

### CODE REVIEW

**Review by IgnatG — APPROVED** [review]

**Review by pbradley-f4 — APPROVED** [review]
:+1:

File: e2e-tests/gef/cypress/integration/submit-to-ukef.spec.js [hunk 1]
```diff
@@ -16,18 +16,17 @@ context('Submit to UKEF', () => {
         cy.apiFetchAllApplications(token);
       })
       .then(({ body }) => {
-        body.items.forEach((item) => {
-          applicationIds.push(item._id);
-        });
+        applicationId = body.items[0]._id;
```
Comment (davehlewis) [inline e2e-tests/gef/cypress/integration/submit-to-ukef.spec.js:19]: The previous test uses the third item in the array - is this a problem?

File: trade-finance-manager-api/src/v1/api.js [hunk 2]
```diff
@@ -499,7 +499,7 @@ const createEstoreFolders = async (eStoreFolderInfo) => {
     });
     return response.data;
   } catch (err) {
-    return err;
+    return false;
```
Comment (davehlewis) [inline trade-finance-manager-api/src/v1/api.js:502]: needs some degree of error handling here it feels like - at least a log

Comment (ttbarnes) [reply trade-finance-manager-api/src/v1/api.js:502]: There is a log but not in this API. This endpoint calls reference data - which then uses estore controller. The Estore controller logs out errors

Comment (davehlewis) [reply trade-finance-manager-api/src/v1/api.js:502]: ok makes sense

File: e2e-tests/gef/cypress/integration/submit-to-ukef.spec.js (hunk 1 above)
Comment (ttbarnes) [reply e2e-tests/gef/cypress/integration/submit-to-ukef.spec.js:19]: I don't think so. The mock deals are almost exactly the same. The indexing in mock data loader is weird.... Probably don't need this change now.. Either way it doesn't seem to make a difference to the test now along with the other changes. I'd vote to not have this as a blocker and get this merged so we can progress other PRs

Comment (davehlewis) [reply e2e-tests/gef/cypress/integration/submit-to-ukef.spec.js:19]: Sounds good to me

**Review by davehlewis — APPROVED** [review]

**Pull request merged**